packages/generator-single-spa/src/generator-single-spa.js

```diff
@@ -9,33 +9,72 @@ module.exports = class SingleSpaGenerator extends Generator {
       type: String,
     })
 
-    this.destinationPath("test")
+    this.option("moduleType", {
+      type: String,
+    })
   }
-  async getFramework() {
-    if (!this.options.framework) {
+  async composeChildGenerator() {
+    let moduleType = this.options.moduleType
+
+    if (!moduleType) {
       const answers = await this.prompt([
         {
-          type: "list",
-          name: "framework",
-          message: "Which framework do you want to use?",
+          type: 'list',
+          name: 'moduleType',
+          message: 'Which kind of in-browser module do you want create-single-spa to create for you?',
```

Comment:
Suggestion: Maybe instead: `Select type to generate`

Reply:
👍 updated